scripts/sct_propseg.py

```diff
@@ -10,16 +10,16 @@
 #
 # About the license: see the file LICENSE.TXT
 #########################################################################################
-from msct_parser import Parser
-import sys
-import sct_utils as sct
 import os
 import shutil
```

Comment:
Is shutil still used? I believe it could be removed.

Reply:
done